src/CabanasRD.sln

```diff
@@ -12,6 +12,9 @@ EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CabanasRD.XUnitTests", "Tests\CabanasRD.XUnitTests\CabanasRD.XUnitTests.csproj", "{E6285AFF-138B-4513-A5FD-93742F7AED62}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "App", "App", "{27826693-7F7E-41EC-8676-42F84C8931D0}"
+	ProjectSection(SolutionItems) = preProject

```

Comment:
What is this?? @RicardoBritoBrens creo que se te fue un archivo para el root del shared project, delete this please!